### PR TITLE
NEW: Display contributors in about dialog

### PR DESCRIPTION
**Pull request by macite**
# Description

Display a list of contributors in the about dialog.

### CODE REVIEW

**Comment by macite** [comment]
@jakerenzella could you have a quick look over this. I think it addresses all of the previous comments from the earlier PR. There could be some styling improvements, but it is functional and I would like to merge it ASAP to close this off.

**Comment by jakerenzella** [comment]
@macite I'll take a look at this tomorrow, I already looked over this and thought it looked a lot better than last time, but just need to run it.

File: src/app/common/modals/about-doubtfire-modal/about-doubtfire-modal.component.ts [hunk 1]
```diff
@@ -73,4 +84,10 @@ export class AboutDoubtfireModal {
         });
     });
   }
+
+  private async getContributorData() {
```
Comment (jakerenzella) [inline src/app/common/modals/about-doubtfire-modal/about-doubtfire-modal.component.ts]: Should probably be using rxjs here for consistency

Comment (macite) [reply src/app/common/modals/about-doubtfire-modal/about-doubtfire-modal.component.ts]: I missed this - it is not needed. So resolved now.

**Pull request merged**